cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -59,6 +57,10 @@ These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of
   actual services, such as the HDFS Namenode, the YARN resource manager, or the HBase
   Master.
 
+- Note that the CDAP Gateway/Router Service is not a `Cloudera Manager Gateway Role 
+  <http://www.cloudera.com/content/www/en-us/documentation/enterprise/latest/topics/cm_mc_managing_roles.html>`__
+  but is instead another name for the CDAP Router Service.
```

Comment:
suggest adding: "CDAP also provides it's own Gateway Role that can be used to install CDAP client configurations on other hosts of the cluster"